src/rdkafka_request.c

```diff
@@ -497,6 +497,59 @@ int rd_kafka_buf_write_topic_partitions(
 }
 
 
+/**
+ * @brief Read current leader from \p rkbuf.
+ *
+ * @param rkbuf buffer to read from
+ * @param CurrentLeader is the CurrentLeader to populate.
+ * @return 1 on success, else -1 on parse error.
```

Comment:
add a newline to separate it from the parameters

Reply:
Updated